can/interfaces/ics_neovi/neovi_bus.py

```diff
@@ -377,8 +371,17 @@ def send(self, msg, timeout=None):
             message.NetworkID = self.channels[0]
         else:
             raise ValueError("msg.channel must be set when using multiple channels.")
+        return message
+
+    def send(self, msgs, timeout=None):
+        self._send_internal(self._get_messages_sequence(msgs), timeout)
+
+    def _send_internal(self, msg, timeout=None):
+        if not ics.validate_hobject(self.dev):
+            raise CanError("bus not open")
+        messages = tuple(self._msg_to_spy_msg(_) for _ in msg)
```

Comment:
Underscore is usually a placeholder for an unused argument, which it is not in this case, right?